.rubocop.yml

```diff
@@ -1,9 +1,16 @@
+# Merge default configurations on Exclude option
+inherit_mode:
+  merge:
+    - Exclude
```

Comment:
RuboCop has default `Exclude` configurations ([link](https://github.com/rubocop/rubocop/blob/v1.69.2/config/default.yml#L65-L69)).
With this option, the default configurations and the configurations in this file are merged.

Reply:
Without this option, RuboCop on CI may fail due to lack of default `Exclude` configurations.

Ref: https://github.com/rubocop/rubocop/issues/9832